apps/admin-x-activitypub/src/views/Preferences/components/EditProfile.tsx

```
@@ -118,8 +133,23 @@ const EditProfile: React.FC<EditProfileProps> = ({account, setIsEditingProfile})
+            let errorMessage = 'Failed to upload image. Try again.';
+
+            if (error && typeof error === 'object' && 'statusCode' in error) {
+                switch (error.statusCode) {
+                case 413:
+                    errorMessage = 'Image size exceeds limit.';
+                    break;
+                case 415:
+                    errorMessage = 'The file type is not supported.';
+                    break;
+                default:
+                    errorMessage = errorMessage;
```

Biome (1.9.4)

[error] errorMessage is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)